src/utils/util.js

```diff
@@ -68,18 +68,46 @@ export const removeResponse = (id) => {
   .then(response => response.json())
 }
 
+export const flagQuestion = (postObject) => {
+  return fetch(`https://developer-mental-health-org.herokuapp.com/api/v1/qflag/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(postObject)
+  })
+  .then(response => response.json())
+  .catch(err => err)
+}
+
+export const flagComment = (postObject) => {
+  return fetch(`https://developer-mental-health-org.herokuapp.com/api/v1/rflag/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(postObject)
+  })
+  .then(response => response.json())
+  .catch(err => err)
+}
+
```

Comment:
These look good - We should plan to go back and do more error handling on the fetch requests sometime this week